src/main/java/uk/gov/hmcts/reform/managecase/service/noc/NoticeOfChangeQuestions.java

```diff
@@ -66,10 +66,16 @@ public ChallengeQuestionsResult getChallengeQuestions(String caseId) {
         ChallengeQuestionsResult challengeQuestionsResult = challengeQuestions(caseId).getChallengeQuestionsResult();
 
         List<ChallengeQuestion> challengeQuestionsResponse = challengeQuestionsResult.getQuestions().stream()
-            .map(challengeQuestion -> {
-                challengeQuestion.setAnswerField(null);
-                return challengeQuestion;
-            })
+            .map(challengeQuestion -> ChallengeQuestion.builder()
+                .caseTypeId(challengeQuestion.getCaseTypeId())
+                .order(challengeQuestion.getOrder())
+                .questionText(challengeQuestion.getQuestionText())
```

Comment:
IMO a separate response class without answer fields better. (separation of layers)
Or move this logic to a method in ChallengeQuestion class #cloneWithoutAnswers(ChallengeQuestion challengeQuestion)